### PR TITLE
ci: Add Semantic Release with GitHub Actions Workflow

### PR DESCRIPTION
**Pull request by deniaz**

### CODE REVIEW

File: .releaserc [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+            "name": "next",
+            "prerelease": true
+        }
+    ],
+    "plugins": [
+        "@semantic-release/commit-analyzer",
+        "@semantic-release/release-notes-generator",
+        "@semantic-release/npm",
+        "@semantic-release/github"
+    ],
```
Comment (SamuelAlev) [inline .releaserc:14]: Does it create a new release on github with the changelog?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+name: Release
+
+on:
+    workflow_dispatch:
+        branches:
+            - "main"
+    push:
+        branches:
+            - "next"
```
Comment (SamuelAlev) [inline .github/workflows/release.yml:9]: We may not want to have push on branch `main` (stable) and `next` (alpha) in the same workflow.
How do we distinguish them? Like `npm i @frontify/arcade@latest` should install a stable release and the `npm i @frontify/arcade@next` should install the alpha.

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+        steps:
+            - name: Checkout default branch
+              uses: actions/checkout@v2
+
+            - name: Use Node.js 16 (npm repository)
+              uses: actions/setup-node@v2
+              with:
+                  node-version: "16"
+                  registry-url: "https://registry.npmjs.org"
```
Comment (SamuelAlev) [inline .github/workflows/release.yml:27]: We are missing the publish on github packages

File: .releaserc (hunk 1 above)
Comment (deniaz) [reply .releaserc:14]: Exactly. :)

**Pull request closed without merging**